src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1378,13 +1395,16 @@ BoundExpression bindDefaultArgument(SyntaxNode syntax, ParameterSymbol parameter
                     string path = callerSourceLocation.SourceTree.GetDisplayPath(callerSourceLocation.SourceSpan, Compilation.Options.SourceReferenceResolver);
                     defaultValue = new BoundLiteral(syntax, ConstantValue.Create(path), Compilation.GetSpecialType(SpecialType.System_String)) { WasCompilerGenerated = true };
                 }
-                else if (callerSourceLocation is object && parameter.IsCallerMemberName)
+                else if (callerSourceLocation is object && parameter.IsCallerMemberName && containingMember is not null)
```

Comment:
>&& containingMember is not null [](http://example.com/codeflow?start=88&length=31)

Are we silently ignoring `[CallerMemberName]` for attributes?

Reply:
We are not ignoring them in batch compilation scenarios, but we are in semantic model (IOperation) scenarios. See https://github.com/dotnet/roslyn/pull/59750#discussion_r814590199